src/lib/viewers/controls/media/MediaSettingsControls/_styles.scss

```diff
@@ -0,0 +1,48 @@
+@import '../styles';
+
+$padding: 8px;
+
+@mixin bp-MediaSettingsMenuItem {
+    display: table-row;
+    height: 26px;
+    padding: $padding / 2;
+    overflow: hidden;
+    white-space: nowrap;
+    outline: 0 none; // will be overridden by our own
```

Comment:
Can we clarify this comment?

Reply:
Copied from the original settings styles... will delete the comment